cranelift/test-zk.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

There are systems that don't store bash at `/bin` (e.g. NixOS.)